projects/packages/my-jetpack/_inc/state/selectors.js

```diff
@@ -10,14 +10,21 @@ export const getProduct = ( state, productId ) => {
 
 	const product = mapObjectKeysToCamel( stateProduct, true );
 	product.pricingForUi = mapObjectKeysToCamel( product.pricingForUi || {}, true );
+	product.pricingForUi.introductoryOffer = product.pricingForUi.isIntroductoryOffer
+		? mapObjectKeysToCamel( product.pricingForUi.introductoryOffer, true )
+		: null;
 	product.features = product.features || [];
 	product.supportedProducts = product.supportedProducts || [];
 
 	product.pricingForUi.fullPricePerMonth =
 		Math.ceil( ( product.pricingForUi.fullPrice / 12 ) * 100 ) / 100;
+	const discountedPricePerMonth =
+		product.pricingForUi.isIntroductoryOffer &&
+		product.pricingForUi.introductoryOffer.intervalUnit === 'month'
```

Comment:
Same here, should we take `intervalCount` into consideration in these checks? Let's be consistent either way.